msteams-platform/tabs/how-to/authentication/tab-sso-overview.md

```diff
@@ -0,0 +1,203 @@
+
+## Obtaining access for app users at runtime
+
+You can achieve SSO in a tab by obtaining access token for the Teams user currently logged in. This process involves the tab app, Microsoft Teams, and Azure AD. During this interaction, the user must consent for use of their Teams identity to obtain access token.
+
+The following image shows how SSO works when a Teams user attempts to access the tab app:
+
+:::image type="content" source="../../../assets/images/authentication/teams-sso-tabs/teams-runtime-seq.png" alt-text="Tab single sign-on SSO diagram" border="false":::
+
+<!--
+```mermaid
+sequenceDiagram
+    User->>Tab: Opens Teams app
+    Tab->>Teams Client: Call getAuthToken()
+    Teams Client->>Sign-in and Consent: Check if consent is required
+    Sign-in and Consent->>Teams Client: Prompt for consent from new user
+    Teams Client->>Azure AD: Request access token from Azure AD
+    Azure AD->>Teams Client: Send access token to Teams client
+    Teams Client->>Tab: Respond to getAuthToken() with access token
+    Tab->>Tab: Parse access token to give access to user
+```
+-->
+
+| # | Interaction | What's going on |
+| --- | --- | --- |
+| 1 | Tab -> Teams | The tab app makes a JavaScript call to `getAuthToken()`, which tells Teams to obtain an access token for the tab application. |
+| 2 | Teams - > Consent form | If the current user is using your tab application for the first time, Teams displays request prompt to consent, if consent is required. The user must provide their consent to Teams for using their Teams identity to obtain access token from Azure AD. <br> Alternately, there's a request prompt to handle step-up authentication such as two-factor authentication. |
+| 3 | Teams -> Azure AD | Teams requests Azure AD endpoint for the tab access token for the current user based on their Teams identity. |
+| 4 | Azure AD -> Teams | Azure AD sends the tab access token to the Teams client. The token is a JSON Web Token (JWT), and it's validation works just like token validation in most standard OAuth flows. Teams will cache the token on your behalf so that future calls to getAccessToken simply return the cached token. |
+| 5 | Teams -> Tab | Teams sends the tab access token to the tab as part of the result object returned by the `getAuthToken()` call. |
+| 6 | Tab app | The tab app parses the token using JavaScript to extract required information, such as the user's email address. The token returned to the tab app is both an access token and an identity token. |
+
+> [!IMPORTANT]
+> The `getAuthToken()` is only valid for consenting to a limited set of user-level APIs that is email, profile, offline_access, and OpenId. It is not used for further Graph scopes such as `User.Read` or `Mail.Read`. For suggested workarounds, see [Extend your app with Microsoft Graph permissions](tab-sso-graph-api.md).
+
+> [!NOTE]
+> Currently, SSO only supports OAuth 2.0 token and does not support SAML token.
```

Comment:
Should we move this to Know limitation section?

Reply:
Moved the content to 'Known limitations'.